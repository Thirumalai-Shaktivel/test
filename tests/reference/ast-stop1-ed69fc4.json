{
    "basename": "ast-stop1-ed69fc4",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/stop1.f90",
    "infile_hash": "356601f3f1d56abfa1b7d00130c2067cbf3d450b3c563a900ff9ccd5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-stop1-ed69fc4.stdout",
    "stdout_hash": "ad565999f862e5779095c4e3c8c34d50313aeef9dcce02b81b33cfb9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}