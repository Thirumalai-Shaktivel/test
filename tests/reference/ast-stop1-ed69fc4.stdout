(TranslationUnit [(Program stop1 () [] [(ImplicitNone [] (TriviaNode [] [(EmptyLines) (Comment "! AST only(Syntax check)")]))] [(Declaration (AttrType TypeInteger [] () None) [] [(stop [] [] 15 Equal ())] ()) (Declaration (AttrType TypeInteger [] () None) [] [(errstop [] [] 25 Equal ())] (TriviaNode [] [(EmptyLines) (EmptyLines)]))] [(If 0 () (/= stop 15) [(Stop 0 () () ())] [(If 0 () (/= errstop 25) [(ErrorStop 0 () () ())] [] () ())] () (TriviaNode [] [(EmptyLines) (EmptyLines)])) (If 0 () (/= stop 15) [(Stop 0 (String "message") () ())] [(If 0 () (/= errstop 25) [(ErrorStop 0 (String "message") () ())] [] () ())] () (TriviaNode [] [(EmptyLines) (EmptyLines)])) (If 0 () (/= stop 15) [(Stop 0 0 () ())] [(If 0 () (/= errstop 25) [(ErrorStop 0 0 () ())] [] () ())] () (TriviaNode [] [(EmptyLines) (EmptyLines)])) (If 0 () (/= stop 15) [(Stop 0 () (Logical .true.) ())] [(If 0 () (/= errstop 25) [(ErrorStop 0 () (Logical .true.) ())] [] () ())] () (TriviaNode [] [(EmptyLines) (EmptyLines)])) (If 0 () (/= stop 15) [(Stop 0 1 (Logical .false.) ())] [(If 0 () (/= errstop 25) [(ErrorStop 0 1 (Logical .false.) ())] [] () ())] () (TriviaNode [] [(EmptyLines) (EmptyLines)]))] [])])
